pb/record.proto

```diff
@@ -16,12 +16,6 @@ message Record {
 	// The actual value this record is storing
 	optional bytes value = 2;
 
-	// hash of the authors public key
-	optional string author = 3;
-
-	// A PKI signature for the key+value+author
-	optional bytes signature = 4;
-
 	// Time the record was received, set by receiver
-	optional string timeReceived = 5;
+	optional string timeReceived = 3;
```

Comment:
please don't change the field id, it is backwards incompatible.

Reply:
(although I'm pretty sure that's *technically* a harmless change.

Reply:
Nice catch.. please take a look at the diff of `record.pb.go`. The protoc compiler I'm using has added in a field called `Descriptor`. I'm not sure what that is so just want to make sure it's ok

Reply:
Never mind I fixed